features/203.IotAgentJson.feature

```diff
@@ -3,8 +3,8 @@ Feature: Test tutorial 203.Why are muliple IoT Agents Needed?
     url: https://fiware-tutorials.readthedocs.io/en/latest/iot-agent-json.html
     git-clone: https://github.com/FIWARE/tutorials.IoT-Agent-JSON.git
     git-directory: /tmp/tutorials.IoT-Agent-JSON
-    shell-commands: ./services create; ./services start
-    clean-shell-commands: ./services stop
+    shell-commands: docker rm fiware-iot-agent ; ./services create; ./services start
```

Comment:
why do you need to execute the docker rm?

Reply:
Because the docker isn't removed by itself. A second execution or other scenarios would fail if the docker is not removed manually.  -- Maybe a comment should be made on the tutorial.

Reply:
Idem to the previous comment -> Issue in services script